### PR TITLE
Update homepage text

### LINKED ISSUE #1140: Home: copy changes
### Hero image text:

MicroMasters: Learn now, apply later (or never)
A new kind of digital credential for online learners

### What is MicroMasters?

The MicroMasters is a new digital credential for online learners.  The MicroMasters credential will be granted to learners who complete an integrated set of graduate-level courses on edX.org and capstone exams.  With the MicroMasters credentials, learners can apply for an accelerated master’s degree program on campus, at MIT or other top universities. 

(optional)
Micromasters is an important project for me. I believe in the model of empowering people.
-- Rafael Reif

#### Online Learning
Take MIT graduate-level courses remotely, from anywhere, and meet other motivated students and professionals  from around the world.

#### Blended Masters
Start a new, faster path to a Master’s degree from MIT or other universities

#### Cost savings
Get a masters degree at a significantly reduced cost

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #1140 
#### What's this PR do?
Update text on the homepage.

screen shot 2016-09-28 at 12 56 41 pm
screen shot 2016-09-28 at 12 57 02 pm

### CODE REVIEW

**Comment by roberthouse54** [comment]
The headers are incorrect. See Peter's comment on #1140. They should have stayed what they were before. I.e.:

What is the MicroMasters program?
Why should I enroll?
Current MicroMasters Programs

Also, the amount of text under "Convenient and Flexible" is a bit long. We should shorten that. Also the header "Convenient and Flexible" is long as well. I prefer "Online Learning" here.

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -39,13 +39,12 @@
...
           <div class="sub-banner-section">
             <div class="sub-banner-text">
-            <h4>What is the MicroMasters program?</h4>
-              <p>The MicroMasters credential is a new option for online learning in which the students can earn
-                a full master's degree by taking about half the course content online and half on campus.</p>
-              <p>The MicroMasters Credential will be granted by MITx (MIT's online learning initiative)
-                to students who do exceptionally well in a set of graduate-level online courses and a subsequent exam.
-                For many learners, the MicroMasters credential can be a step toward earning an MIT master's degree at a
-                great cost saving.</p>
+            <h4>What is?</h4>
```
Comment (Ferdi) [inline cms/templates/cms/home_page.html]: Change the title to "What is MicroMasters?"

File: cms/templates/cms/home_page.html [hunk 2]
```diff
@@ -60,30 +59,31 @@
 
       <div class="centered-content">
 
-        <h3>Why should I enroll?</h3>
+        <h3>Why?</h3>
```
Comment (Ferdi) [inline cms/templates/cms/home_page.html]: "Why should I enroll?"

**Comment by Ferdi** [comment]
I agree with @roberthouse54

**Comment by annagav** [comment]
Should have the right headers now! Yesterday Rob convinced me that the headers change as well :D

**Comment by annagav** [comment]
screen shot 2016-09-28 at 1 50 05 pm

File: cms/templates/cms/home_page.html (hunk 1 above)
Comment (Ferdi) [inline cms/templates/cms/home_page.html]: @annagav can you change ?

File: cms/templates/cms/home_page.html [hunk 3]
```diff
@@ -67,23 +66,24 @@
           </div>
 
           <h5 class="upper-case">Online Learning</h5>
-          <p>Take graduate-level courses remotely, from anywhere, and meet other students from around the world.</p>
+          <p>Take MIT graduate-level courses remotely, from anywhere, and meet
```
Comment (Ferdi) [inline cms/templates/cms/home_page.html]: Change the text to 
"Take MIT graduate-level courses remotely, from anywhere in the world."

**Comment by annagav** [comment]
screen shot 2016-09-28 at 2 16 21 pm

**Comment by pdpinch** [comment]
Nevermind. Rob has https://github.com/mitodl/micromasters/pull/1155#pullrequestreview-2001608 in another PR

**Pull request merged**